ysh/grammar.pgen2

```diff
@@ -597,19 +597,15 @@ regex: [re_alt] (('|'|'or') re_alt)*
 case_pat: (
```

Comment:
The comment above this with ^^^^^^ seems a bit confusing, since `(0)` is is also a pattern